wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -380,7 +389,7 @@ final class MLSServiceTests: ZMConversationTestsBase, MLSServiceDelegate {
         // Given
         let groupID = MLSGroupID(Data([1, 2, 3]))
         let config = ConversationConfiguration(
-            ciphersuite: CiphersuiteName.mls128Dhkemx25519Aes128gcmSha256Ed25519.rawValue,
+            ciphersuite: UInt16(self.defaultCipherSuite.rawValue),
```

Comment:
should we change the type of ciphersuite in ConversationConfiguration to Int?

Reply:
`ConversationConfiguration` comes from CC